shoe store/index.html

```diff
@@ -0,0 +1,133 @@
+                
+                <p>Customer satisfaction is our top priority. We strive to provide an exceptional shopping experience, from the moment you browse our website to the delivery of your new favorite pair of shoes. Our knowledgeable staff is here to assist you every step of the way, whether you need help finding the perfect fit or have questions about our products.</p>
+                
+                <p>Thank you for choosing Shoe Store. We look forward to helping you step out in style!</p>
+            </section>
+            <span class="top-trending">Top Trending</span>
+<div class="showroom">
+
+    <div class="card">
+        <img src="assets/shoe1.jpg" alt="Placeholder Image">
+        <h3>Nike Air Max</h3><p></p>
+        <p>The Nike Air Max is a classic athletic shoe known for its comfort and style. It features a cushioned sole and breathable materials, making it ideal for running and everyday wear.</p>
+    </div>
+    <div class="card">
+        <img src="assets/shoe2.jpg" alt="Placeholder Image">
+        <h3>Adidas Superstar</h3>
+        <p>The Adidas Superstar is an iconic sneaker with a timeless design. It features a leather upper, rubber shell toe, and signature three stripes, making it a versatile choice for casual outfits.</p>
+    </div>
+    <div class="card">
+        <img src="assets/shoe3.jpg" alt="Placeholder Image">
+        <h3>Vans Old Skool</h3>
+        <p>The Vans Old Skool is a popular skate shoe known for its durability and classic style. It features a suede and canvas upper, padded collar, and signature side stripe, making it perfect for skateboarding and everyday wear.</p>
+    </div>
+    <div class="card">
+        <img src="assets/shoe4.jpg" alt="Placeholder Image">
+        <h3>Converse Chuck Taylor All Star</h3>
+    <p>The Converse Chuck Taylor All Star is a legendary sneaker that has been worn by generations. It features a canvas upper, rubber toe cap, and iconic star logo, making it a timeless choice for casual attire.</p>
+    </div>
+    <div class="card">
+        <img src="assets/shoe5.jpg" alt="Placeholder Image">
+       
+    <h3>New Balance 574</h3>
+    <p>The New Balance 574 is a versatile running shoe with a retro-inspired design. It features a suede and mesh upper, ENCAP midsole technology, and durable rubber outsole, providing comfort and support for all-day wear.</p>
+    </div>
+    <div class="card">
+        <img src="assets/shoe6.jpg" alt="Placeholder Image">
+        <h3>Puma Suede Classic</h3>
+        <p>The Puma Suede Classic is a stylish sneaker with a rich heritage. It features a suede upper, padded collar, and classic Formstrip branding, making it a must-have for fashion-forward individuals.</p>
+    </div>
+</div>
```

Comment:
- [ ] Let's improve the html structure by including the **Top Trending** within the `showroom` div and wrapping all the cards within a `cards-wrapper` div that should be a flex container.